src/components/nav-bar.js

```diff
@@ -0,0 +1,40 @@
+import { html, LitElement } from "lit-element";
+import { connect } from "pwa-helpers";
+
+import { store } from "../redux/store.js";
+import { logout } from "../redux/login-actions.js";
+
+class NavBar extends connect(store)(LitElement) {
+  static get properties() {
+    return {
+      isLoggedIn: {
+        type: Object,
+      },
+    };
+  }
+
+  stateChanged(state) {
+    this.isLoggedIn = state.login.user;
+  }
+
+  logout() {
+    store.dispatch(logout());
+    window.location.href = "http://localhost:8080/";
```

Comment:
Always use values like this in config. Also, try to use url path without hostname like if you want to go to home use "/" instead of whole hostname. Use Router.go for such purpose.